README.md

```diff
@@ -18,16 +18,6 @@ Add the following line to your `Gemfile`:
 gem "merch_calendar"
 ```
 
-
-## Configuration
-```ruby
-# NOTE: Configuration will be added soon, but is currently NOT available.
-MerchCalendar.configure do |config|
-  # The month that Q1 begins. The default is 8 (August)
-  config.quarter_start_month = 8
-end
-```
-
```

Comment:
i know it wasn't part of your changes but, on line 8, could you make Stitch Fix two words?